changelog/undistributed/changelog_iosxr_show_bgp_instance_all_all_all_20210808.rst

```diff
@@ -0,0 +1,11 @@
+--------------------------------------------------------------------------------
+                            Fix
+--------------------------------------------------------------------------------
+* IOSXR
+    * Modified ShowBgpInstanceAllAllSchema:
+        * Modified key 'local_as' to Or(int, str). This captures dotted Notation ASN which is string.
+    * Modified ShowBgpInstanceAllAll:
+        * Modified RegEx <p6>,<p16_2>,<p16>,<m3> under <p16>, <p17> to capture dotted Notation ASN in BGP
+    * Modified UT test_show_bgp.py to test output with dotted ASN
+    * Added UT folder for ShowBgpInstanceAllAll for output with dotted ASN
```

Comment:
No need to mention unit tests in the changelog

Reply:
removed from changelog